bin/critterget.py

```diff
@@ -601,7 +645,7 @@ def getCredentials(sessionKey):
             if auth is None:
                 print u'{} MessageType="ApteligentDebug" No credentials have been found for app {} . Maybe a setup issue?'.format(DATETIME_OF_RUN, myapp)
         retry += 1
-        time.sleep(5)
+        time.sleep(10)
```

Comment:
I tried a few values and settled on 10 secs as the best wait time to minimize the chance that the integration will fail to get its credentials during its first attempt and force me to wait 10 min before it tries again.